frame/composable-traits/src/vault.rs

```diff
@@ -27,6 +28,18 @@ pub enum FundsAvailability<Balance> {
 	None,
 }
 
+#[derive(Copy, Clone, Encode, Decode, MaxEncodedLen, Default, Debug, PartialEq, TypeInfo)]
```

Comment:
Let's go ahead and add `Eq` to this while making changes.

Reply:
Resolved